### DISCUSSION TITLE
OMP breaks when using screen

### DISCUSSION THREAD

**Discussion by aagha** [JanDeDobbeleer/oh-my-posh#3761] [Troubleshoot] [answered]
When using `screen`, my prompt seems to break:

Normal:
![image](https://user-images.githubusercontent.com/5522103/234411205-9a1a8e73-bd8f-4c17-be1b-bb2e866cc490.png)

Screen:
![image](https://user-images.githubusercontent.com/5522103/234411213-b8a8b009-8d9f-4f1c-a33b-b92afe47c341.png)

Things seem to be fine with tmux though:
![image](https://user-images.githubusercontent.com/5522103/234411288-ddb09090-50d4-4890-b05b-fcc805d5557c.png)

Any idea why it break with screen and how to fix it?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@aagha this has been [reported before](https://github.com/JanDeDobbeleer/oh-my-posh/discussions/556) but we have no solution for this. Seems like somehow screen doesn't respect the ANSI coloring (which it should).
